src/styles/blocks/catalog.scss

```
@@ -17,6 +17,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transform: 300ms;
+    transition: 300ms;
+
+    &:hover {
+      transform: scale(1.2);
+      background-color: #34568b;
```

```suggestion
      color: #34568b;
```

and you can also move color to variable